src/main/resources/okta/js/applications.js

```diff
@@ -0,0 +1,618 @@
+;(function( $ ) {
+    $.fn.zclip = function() {
+        if(typeof ZeroClipboard == 'function'){
+            var client = new ZeroClipboard( this );
+            client.on( "ready", function( readyEvent ) {
+              client.on( "aftercopy", function( event ) {
+                var target = $(event.target);
+                target.attr("title","Copied!")
+                target.tooltip('enable');
+                target.tooltip("show");
+                target.tooltip('disable');
+              });
+            });
+        }else{
+            console.warn('Warning : Dependency missing - ZeroClipboard Library');
+        }
+        return this;
+    };
+}( jQuery ));
+
+
+var GrantTypes = function (available) {
+    //order will be preserved in the response map
+    this.config = {
+        "authorization_code":"Code",
+        "implicit":"Implicit",
+        "refresh_token":"Refresh Token", 
+        "password":"Password", 
+        "iwa:ntlm":"IWA-NTLM", 
+        "client_credentials":"Client Credential", 
+        "urn:ietf:params:oauth:grant-type:saml2-bearer":"SAML2",
```

Comment:
We can add dynamic grant types in identity.xml. How will they be handled at this point?

Reply:
This is from production code
 https://github.com/wso2/carbon-apimgt/blob/v6.1.66/features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/js/applications.js#L24